src/iuse.h

```diff
@@ -80,6 +80,11 @@ std::optional<int> call_of_tindalos( Character *, item *, const tripoint & );
 std::optional<int> camera( Character *, item *, const tripoint & );
 std::optional<int> can_goo( Character *, item *, const tripoint & );
 std::optional<int> capture_monster_act( Character *, item *, const tripoint & );
+std::optional<int> heat_solid_items( Character *p, item *it, const tripoint & );
+std::optional<int> heat_liquid_items( Character *p, item *it, const tripoint & );
+std::optional<int> heat_all_items( Character *p, item *it, const tripoint & );
+std::optional<int> heat_items( Character *p, item *it, bool liquid_items, bool solid_items );
+std::optional<int> heat_single_item( Character *p, item *it );
```

Comment:
These are helper functions - they should be `static` functions with `iuse.cpp`, and only return `std::optional<int>` if they need to.